.github/workflows/sync-upstream-change.yml

```diff
@@ -12,11 +17,26 @@ jobs:
       github.event.review.state == 'changes_requested' &&
       github.event.review.body == 'bot change'
     steps:
+      - name: Retrieve secrets
+        uses: grafana/shared-workflows/actions/get-vault-secrets@main
+        with:
+          repo_secrets: |
+            APP_ID=app:id
+            APP_PRIVATE_KEY=app:private-key
+
+      - name: Generate a token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
```

Comment:
Use a pinned version please (and the others)

Reply:
Oh right!

Reply:
Done!